backend/inspirehep/search/facets.py

```diff
@@ -80,6 +80,28 @@ def inner(values):
     return inner
 
 
+def hep_subject_must_match_all_or_missing_filter(field):
+    """Bool filter containing a list of must matches."""
+
+    def inner(values):
+
+        if current_app.config.get("SUBJECT_MISSING_VALUE", "Unknown") in values:
```

Comment:
`Unknown` is already the default value of the `SUBJECT_MISSING_VALUE`, isn't it?